### PR TITLE
 Use cloud native network security tools and controls.md

### PR DESCRIPTION
**Pull request by yelghali**
@dubrie

### CODE REVIEW

File: docs/catalog/cloud/Use cloud native network security tools and controls.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- security
```
Comment (dubrie) [inline docs/catalog/cloud/Use cloud native network security tools and controls.md:7]: please include "cloud" and "network" to the list of tags as well.

File: docs/catalog/cloud/Use cloud native network security tools and controls.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- compute
+- kubernetes
+- role:software-engineer
+- size:medium
+---
+
+# Use cloud native network security tools and controls
+
+## Description
+Network & Web Application Firewalls provide protection against OWASP top 10 attacks and load shedding bad bots. Using these capabilities helps remove unnecessary data transmission and reduces the burden on the cloud infrastructure, with lower bandwidth and less infrastructure requirements.
```
Comment (dubrie) [inline docs/catalog/cloud/Use cloud native network security tools and controls.md:17]: What does OWASP stand for in this context?

File: docs/catalog/cloud/Use cloud native network security tools and controls.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Solution
+Filter Ingress and Egress traffic at the source, to remove the unnecessary data transmissions.
```
Comment (dubrie) [inline docs/catalog/cloud/Use cloud native network security tools and controls.md:20]: How do you define an "unnecessary" data transmission?

**Review by dubrie — CHANGES_REQUESTED** [review]
@yelghali changes requested from the initial review

**Comment by yelghali** [comment]
@dubrie thank you ok

**Review by dubrie — APPROVED** [review]

**Comment by dubrie** [comment]
Looks good, thank you!

**Pull request merged**